src/components/experiment-tracking/run-dataset/run-dataset.js

```diff
@@ -134,6 +163,7 @@ const RunDataset = ({
  * @param {function} setShowRunPlotsModal callbak function to show runplot modal
  */
 function buildDatasetDataMarkup(
+  index,
```

Comment:
Are you using this anywhere? It doesn't seem like you are and if not you can remove it.

Reply:
good spot, i dont think i need to use it anymore. 👍